parsing/lexer.mll

```diff
@@ -290,6 +290,9 @@ let identchar_latin1 =
   ['A'-'Z' 'a'-'z' '_' '\192'-'\214' '\216'-'\246' '\248'-'\255' '\'' '0'-'9']
 let symbolchar =
   ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '|' '~']
+let quoted_string_id_char =
+  lowercase |
+      ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '~']
```

Comment:
No particular reason, I just took the existing symbolchars characters as a reasonable starting set. I can respin with '#' if that's wanted.